starter/faq.md

```diff
@@ -17,8 +17,8 @@ Routes and other application-specific logic may live in as many files
 as you wish, in any directory structure you prefer. View the following
 examples for inspiration:
 
-* [Route listings](https://github.com/strongloop/express/blob/master/examples/route-separation/index.js#L19)
-* [Route map](https://github.com/strongloop/express/blob/master/examples/route-map/index.js#L47)
+* [Route listings](https://github.com/strongloop/express/blob/2ac25098548f739c4f2b526b2a00aa60a74c8e75/examples/route-separation/index.js#L32-L47)
```

Comment:
Oh, these links are good to change, since they could stale at any moment. Could you perhaps change them to be based off a git tag rather than a tree hash? Then it would be easy to identify it needs to be updated.